packages/debian/README.md

```diff
@@ -0,0 +1,31 @@
+<h1>
+    <img src="artwork/generated/metalk8s-logo-wide-black-400.png" width="400" height="100%" alt="MetalK8s logo" title="Debian packages" />
+</h1>
+
+## Requirement
+
+To build mannually Debian packages, install `devscripts`, `debhelper`
+and `buid-essential`:
```

Comment:
Typo, and let's have the commands with a helpful package name since we don't all run Ubuntu.

`to build manually, the commands dh and debuild are required on top of the MetalK8s build requirements.` or something like that